src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.windows.cs

```diff
@@ -215,10 +215,9 @@ public void Dispose()
 
 	protected virtual partial void PlatformUpdateShouldKeepScreenOn()
 	{
-		if (MediaElement.ShouldKeepScreenOn)
+		if (MediaElement is not null && MediaElement.ShouldKeepScreenOn)
```

Comment:
I think you could change this to:
```suggestion
		if (MediaElement?.ShouldKeepScreenOn is true)
```
 But happy for you to keep as is if you prefer how it reads